### PR TITLE
THORN-2356: MicroProfile Open API 1.1 test @PATCH, @Content example, @Extension

### PR DESCRIPTION
**Pull request by penehyba**

### CODE REVIEW

**Comment by juangon** [comment]
@penehyba so is this a draft? Should I wait for more commits?

File: microprofile/microprofile-openapi-1.1/src/test/java/org/wildfly/swarm/ts/microprofile/openapi/v11/MicroProfileOpenApi11Test.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import org.junit.runner.RunWith;
+import org.wildfly.swarm.jaxrs.JAXRSArchive;
+import org.yaml.snakeyaml.Yaml;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(Arquillian.class)
+public class MicroProfileOpenApi11Test {
+    @Deployment(testable = false)
+    public static Archive<?> createDeployment() {
```
Comment (Ladicek) [inline microprofile/microprofile-openapi-1.1/src/test/java/org/wildfly/swarm/ts/microprofile/openapi/v11/MicroProfileOpenApi11Test.java]: `@DefaultDeployment` doesn't work?

**Comment by Ladicek** [comment]
Otherwise LGTM.

File: microprofile/microprofile-openapi-1.1/src/test/java/org/wildfly/swarm/ts/microprofile/openapi/v11/MicroProfileOpenApi11Test.java (hunk 1 above)
Comment (penehyba) [reply microprofile/microprofile-openapi-1.1/src/test/java/org/wildfly/swarm/ts/microprofile/openapi/v11/MicroProfileOpenApi11Test.java]: fixed

**Pull request merged**